channels/1ds-post-js/src/DataModels.ts

```diff
@@ -262,6 +262,16 @@ export interface IChannelConfiguration {
      * @since 3.3.1
      */
     fetchCredentials?: RequestCredentials;
+
+    /*
+     * [Optional] If set to false and the CompressionStream API is available, the payload will be compressed using the CompressionStream API.
```

Comment:
Lets change the wording a little.

[Optional] If not set to `true` and the CompressionStream API...